code/__HELPERS/mobs.dm

```diff
@@ -473,3 +473,31 @@ GLOBAL_LIST_EMPTY(species_list)
 		chosen = pick(mob_spawn_meancritters)
 	var/mob/living/simple_animal/C = new chosen(spawn_location)
 	return C
+
+/**
+  * Gives the target carbon mob their items listed in species.items_important_for_life
+  *
+  * Checks all items worn/held/in inventory on mob and gives/equips if they don't already have it.
+  * Tries to equip, put in hands, or drop underneath in that order.
+  */
+/proc/give_important_for_life(var/mob/living/carbon/target)
+	// Path to all items classed as important for a mob to live
+	var/list/important_for_life = target.dna.species.items_important_for_life

```

Comment:
Not all carbons have dna and/or species. Not sure if it's causing issues here, but it might if you are trying to abduct monkeys.

Reply:
Yeah good catch.

Reply:
thats why this should also be a species proc